sklearn/decomposition/_pca.py

```diff
@@ -210,6 +210,12 @@ class PCA(_BasePCA):
 
         .. versionadded:: 0.18.0
 
+    n_oversamples_rate : float, default=0.8
+        The percentage of features that need to be extract.
+        Must be of range [0, infinity).
+
+        .. versionadded:: 0.18.0
```

Comment:
```suggestion
        .. versionadded:: 1.1
```

Reply:
I have completed the modification and submitted it, please check.